src/components/games/seabattle/game.ts

```diff
@@ -150,14 +160,14 @@ function checkAllShipsSunk(ships: IShip[], player: number): boolean {
 }
 
 function randomlyGetShip(player: number, shipSize: number): IShip {
-  const cell: ICell = {x: getRandomInt(10), y: getRandomInt(10)};
+  const cell: ICell = { x: getRandomInt(10), y: getRandomInt(10) };
```

Comment:
were these changes made by the linter? if they werent, avoid making random changes that dont impact the code.

Reply:
They were auto-formatted by my setup and I didn't realize.  I reverted the file back to the old formatting.  At some point in the future I may create a PR for formatting.